src/util/mt_inotify.cc

```diff
@@ -127,7 +127,7 @@ struct inotify_event* Inotify::nextEvent()
     // first_byte is index into event buffer
     if (firstByte != 0
         && firstByte <= int(bytes - sizeof(struct inotify_event))) {
-        ret = reinterpret_cast<struct inotify_event*>(reinterpret_cast<char*>(event.data()) + firstByte);
+        std::memcpy(ret, reinterpret_cast<char*>(event.data()) + firstByte, sizeof(inotify_event));
```

Comment:
There is another `copy_n` in the code below. We should only use one of `copy_n` or `memcpy`

Reply:
Probably the latter since it's faster (copy_n uses memmove).